### PR TITLE
eslint-config: Turn on flowtype/require-readonly-react-props

### PR DESCRIPTION
**Pull request by tbergq**
It seems like it is finally working

### CODE REVIEW

File: src/eslint-config-adeira/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - Remove `__` global, legacy of Nitrolib from Kiwi.com, see: https://kiwicom.github.io/nitrolib/services.html#intl
+- Set `flowtype/require-readonly-react-props` to NEXT_VERSION_ERROR
```
Comment (mrtnzlml) [inline src/eslint-config-adeira/CHANGELOG.md:4]: `NEXT_VERSION_ERROR` is quite an internal thingy. I usually write something like "warnings or errors in strict mode" for the public. Would you please consider it?

Comment (tbergq) [reply src/eslint-config-adeira/CHANGELOG.md:4]: Yes, you are right, I will update it

Comment (tbergq) [reply src/eslint-config-adeira/CHANGELOG.md]: 🚀

**Review by mrtnzlml — APPROVED** [review]
I am wondering: is it working now OR we are not using any of the cases mentioned here? https://github.com/gajus/eslint-plugin-flowtype/pull/400#issuecomment-492828739

**Comment by tbergq** [comment]
> I am wondering: is it working now OR we are not using any of the cases mentioned here? [gajus/eslint-plugin-flowtype#400 (comment)](https://github.com/gajus/eslint-plugin-flowtype/pull/400#issuecomment-492828739)

I tested with `props: $FlowFixMe` and it didn't produce any error. I didn't specifically try for the rest though. 

But they were for relay things, right? And we should have gotten errors for that in `example-relay`?

**Comment by mrtnzlml** [comment]
@tbergq There is a chance that when I was trying it, we had much more React code in Universe. Perhaps `src/__flowtests__` would be a good place to verify these things?

But not that big deal - 🚀

**Comment by tbergq** [comment]
> @tbergq There is a chance that when I was trying it, we had much more React code in Universe. Perhaps `src/__flowtests__` would be a good place to verify these things?
> 
> But not that big deal - 🚀

Yeah, possibly there was more react code here. I tested in in `kiwicom/account` and no unexpected errors there either

**Pull request merged**